### PR TITLE
Added hp.uniformint to the search space docs.

### PR DESCRIPTION
**Pull request by sebastiaanvergunst**

### CODE REVIEW

**Comment by marload** [comment]
Nice Work!

File: docs/templates/getting-started/search_spaces.md [hunk 1]
```diff
@@ -73,6 +73,10 @@ The stochastic expressions currently recognized by hyperopt's optimization algor
 * `hp.uniform(label, low, high)`
   * Returns a value uniformly between `low` and `high`.
   * When optimizing, this variable is constrained to a two-sided interval.
+  
+* `hp.uniformint(label, low, high)`
+  * Returns an integer uniformly between `low` and `high`.
```
Comment (marctorsoc) [inline docs/templates/getting-started/search_spaces.md:78]: are low and high included?

Comment (marctorsoc) [reply docs/templates/getting-started/search_spaces.md:78]: I think hp.uniformint(label, high) is also valid, and then would between 0 and high. Can you please check and rewrite this?

**Comment by marctorsoc** [comment]
any progress on this @marload ?

**Comment by maxpumperla** [comment]
@sebastiaanvergunst can you please have a look at @marctorsoc 's comment? I want to get this merged.

**Comment by marctorsoc** [comment]
if anyone else wants to take this and end it, go for it :)

**Pull request closed without merging**